src/Shared/Controls/DebugLogs.xaml

```diff
@@ -0,0 +1,116 @@
+                </TextBlock>
+            </labs:SettingsCard.Content>
+        </labs:SettingsCard>
+
+        <labs:SettingsCard
+			ActionIcon="{ui:SymbolIcon Symbol=Clear}"
+			Header="Delete User Data"
+			HeaderIcon="{ui:SymbolIcon Symbol=Delete}"
+			IsClickEnabled="True">
+            <interactivity:Interaction.Behaviors>
+                <core:EventTriggerBehavior EventName="Click">
+                    <core:InvokeCommandAction Command="{x:Bind AppRoot.AppDebug.DeleteUserDataCommand, Mode=OneWay}"/>
+                </core:EventTriggerBehavior>
+            </interactivity:Interaction.Behaviors>
+        </labs:SettingsCard>
+
+        <labs:SettingsCard
+			Header="Memory"
+			IsClickEnabled="True" >
+            <labs:SettingsCard.Content>
+                <TextBlock >
+                    <Run Text="Memory Used:"/>
+                    <Run Text="{x:Bind AppRoot.AppDebug.MemoryUsage}"/>
+                    <Run Text="Expected Memory Limit:"/>
+                    <Run Text="{x:Bind AppRoot.AppDebug.TotalMemory}"/>
+                </TextBlock>
+            </labs:SettingsCard.Content>
+        </labs:SettingsCard>
+
+        <labs:SettingsCard
+			ActionIcon="{ui:FontIcon Glyph=&#xE8A7;}"
+			Header="Open logs folder"
+			HeaderIcon="{ui:SymbolIcon Symbol=Folder}"
+			IsClickEnabled="True">
+            <interactivity:Interaction.Behaviors>
+                <core:EventTriggerBehavior EventName="Click">
+                    <core:InvokeCommandAction Command="{x:Bind AppRoot.AppDebug.OpenLogFolderCommand, Mode=OneWay}"/>
+                </core:EventTriggerBehavior>
+            </interactivity:Interaction.Behaviors>
+        </labs:SettingsCard>
+
+        <TextBlock
+			Margin="1,28,0,4"
+			Style="{StaticResource BodyStrongTextBlockStyle}"
+			Text="Logs" />
+
+        <ScrollViewer x:Name="PART_SvLogs"
+                      MaxWidth="1600"
+                      MaxHeight="450" HorizontalScrollBarVisibility="Visible" HorizontalScrollMode="Enabled" >
+            <ListView  ItemsSource="{x:Bind AppRoot.AppDebug.AppLogs, Mode=OneWay}" >
+
+                <ListView.ItemsPanel>
+                    <ItemsPanelTemplate>
+                        <VirtualizingStackPanel>
+                            <VirtualizingStackPanel.ChildrenTransitions>
+                                <TransitionCollection/>
+                            </VirtualizingStackPanel.ChildrenTransitions>
+                        </VirtualizingStackPanel>
+                    </ItemsPanelTemplate>
+                </ListView.ItemsPanel>
+
+                <ListView.ItemTemplate>
+                    <DataTemplate>
+                        <StackPanel >
+                            <TextBlock Text="{Binding }" TextWrapping="WrapWholeWords"/>
+                            <TextBlock Text="-----------------------------------------------------------------------------------------------------------------------------------------------"/>
```

Comment:
Please don't use text to emulate UI elements.

We also shouldn't need a separator for every log message. If the log is hard to read, we should tweak the style to compensate.

```suggestion
```